wire_protocol/src/commands.rs

```diff
@@ -28,7 +28,7 @@ pub const OLDEST_COMPATIBLE_VERSION: i32 = 5;
 pub const TYPE_SIZE: u32 = 4;
 pub const TYPE_PLUS_LENGTH_SIZE: u32 = 8;
 pub const MAX_WIRECOMMAND_SIZE: u32 = 0x00FF_FFFF; // 16MB-1
-
```

Comment:
The wire_version  field needs to be updated to reflect which wire-commands are being included. (Check to be sure what changes have happened between 11 and 15 in Java and make any updates here.)

Reply:
CreateTransientSegment is added under `WIRE_VERSION` of 14. But rust client is missing `GetTableSegmentInfo` under `WIRE_VERSION` of 12 and `FlushToStorage` under `WIRE_VERSION` of 12. How to proceed?

Reply:
Done